app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NotificationManager.java

```diff
@@ -14,15 +14,15 @@
 import org.mozilla.vrbrowser.ui.views.UIButton;
 import org.mozilla.vrbrowser.ui.widgets.NotificationManager.Notification.NotificationPosition;
 
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 
 public class NotificationManager {
 
     private static final int DEFAULT_DURATION = 3000;
 
-    private static HashMap<Integer, NotificationData> mData = new HashMap<>();
+    private static ConcurrentHashMap<Integer, NotificationData> mData = new ConcurrentHashMap<>();
```

Comment:
Why the need for a `ConcurrentHashMap`?

Reply:
We don't really. We were iterating over the Hashmap and then calling `hide` which removed the items using `remove` on the Hashmap which causes a `concurrentmodificationexception`. So it was a lazy way of reusing that method. I've pushed a PR to avoid using the `ConcurrentHashMap`: https://github.com/MozillaReality/FirefoxReality/pull/2847